src/pages/ProductDetail/ProductDetailMain/ProductCount/ProductCount.jsx

```diff
@@ -2,7 +2,7 @@ import React, { useState } from "react";
 import { BiPlus, BiMinus, BiWon } from "react-icons/bi";
 import "./ProductCount.scss";
 
-function ProductCount() {
+function ProductCount({ price }) {
```

Comment:
- 이 카운트 아래에 plus함수에 대한 로직에 대해 간단히 말씀드리자면
```suggestion
 const plus = () => {
    if (count >= 10) {
      alert("최대 10개까지 구매 가능합니다.");
      return;
    }
    setCount(count + 1);
  };
```
- 이런식으로도 진행할 수 있겠습니다